### PR TITLE
Add platform tools API documentation

### PR DESCRIPTION
**Pull request by jfknoepfli**
- Add API documentation
- Add release notes entry

### CODE REVIEW

File: apis/en/source/pages/machineimage/machineImage_platformTools.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+.. function:: GET /users/{uid}/appliances/{aid}/images/platformtools/{tfid}
+
+.. sidebar:: Summary
+
+	* Method: ``GET``
+	* Response Code: ``200 / 304``
+	* Response Formats: ``application/xml`` ``application/json``
+	* Since: ``UForge 3.8.3``
+
+Retrieves platform tools that matches to the appliance and image format.
```
Comment (emuus) [inline apis/en/source/pages/machineimage/machineImage_platformTools.rst]: Retrieves the platform tools that match the appliance and image format.

File: apis/en/source/pages/machineimage/overview.rst [hunk 2]
```diff
@@ -27,6 +27,7 @@ Machine Image API Resources
 	* :ref:`machineImage-generate`: Request to generate a machine image from an appliance
 	* :ref:`machineImage-get`: Retrieves the meta-data of a machine image
 	* :ref:`machineImage-getAll`: Returns all of the machine images of an appliance
+	* :ref:`machineImage-platformTools`: Retrieves platform tools that matches to the appliance and image format
```
Comment (emuus) [inline apis/en/source/pages/machineimage/overview.rst]: Retrieves platform tools that match the appliance and image format

File: apis/en/source/pages/scanImage_platformTools.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+.. function:: GET /users/{uid}/scannedinstances/{siid}/scans/{sid}/images/platformtools/{tfid}
+
+.. sidebar:: Summary
+
+	* Method: ``GET``
+	* Response Code: ``200 / 304``
+	* Response Formats: ``application/xml`` ``application/json``
+	* Since: ````
+
+Retrieves platform tools that matches to the scan image.
```
Comment (emuus) [inline apis/en/source/pages/scanImage_platformTools.rst]: Retrieves platform tools that match the scan image.

File: changelog.rst [hunk 4]
```diff
@@ -10,6 +10,7 @@ Release Date: 2018-04-16
 
 Enhancements
 ~~~~~~~~~~~~
+* Visualise all the software (libraries, drivers or packages) that are automatically injected by UForge during the generation of a machine image for a specific target cloud environment.
```
Comment (emuus) [inline changelog.rst]: Visualize all .... that is automatically

**Comment by jfknoepfli** [comment]
Thanks. I will do the necessary updates.

**Pull request merged**